src/store/reducers/authReducer/auth.reducer.ts

```diff
@@ -0,0 +1,72 @@
+import { AnyAction } from 'redux';
+import { types } from '../../actions';
+
+const initialState = {
+  loginSuccessful: null as boolean | null,
```

Comment:
please define the type of the initial state rather than casting the values

Reply:
sure ma'am